pkg/platform/aws/template.go

```diff
@@ -104,11 +104,70 @@ module "aws-{{.Config.ClusterName}}" {
   {{- end }}
 }
 
+{{ range $index, $pool := .Config.WorkerPools }}
+module "worker-pool-{{ $index }}" {
+  source = "../lokomotive-kubernetes/aws/flatcar-linux/kubernetes/workers"
+
+  providers = {
```

Comment:
I wonder if this is still needed, as now our modules does not declare the providers, only the requirements. Can we try to remove it?

Reply:
without it, I keep getting the error "Region argument missing"

Reply:
ah :+1: all good then. We can investigate that another time then :)